### DISCUSSION TITLE
Email to be used when applying

### DISCUSSION THREAD

**Discussion by ryankuayj** [community/community#126786] [GitHub Education]
### Hi everyone,

Must our academic email be the primary email when applying for the benefits? I got rejected with the following reason:

- If your school provides an academic email, then please verify and use your academic email address. If your school does not provide an academic email address, then you must enable use of your device camera to capture your image.

even though I have already added my academic email address as my secondary email address.

**Comment by davevad93** [comment]
Hi @NO-TOUCHYTOUCHY ,

Check that your academic email is verified on GitHub, if it's verified it should appear on the application page to receive your student benefits. If it still doesn't appear, set your school email as your primary email in the settings, although I believe you should be able to submit your application if your academic email was verified whether it's your primary email or not.
